app/services/placements/partnerships/notify/create.rb

```diff
@@ -6,7 +6,7 @@ class Create < Notify
 
         def notify_users
           partner_organisation_users.each do |user|
-            UserMailer.with(service: :placements)
+            ::Placements::UserMailer
```

Comment:
Any reason why we have `Claims::UserMailer` but `::Placements::UserMailer` is prefixed with `::`?

I wonder why it works fine in one place but not the other.